src/reconstruction/community.jl

```diff
@@ -253,7 +253,7 @@ function join_with_exchanges(
         throw(
             DomainError(
                 "biomass_ids",
-                "Please add supply the string ids of the biomass functions when
+                "Please supply the string ids of the biomass functions when
```

Comment:
what about squashing the 2 arguments into a single list `objective_biomass_ids` ?

(also, `nothing` vs `[]` may apply)

Reply:
ah yes, that would simplify the function further!